packages/driver-stylelint/src/StylelintDriver.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs';
+import { ConfigContext, Driver, Execution, Path } from '@beemo/core';
+import { Event } from '@boost/event';
+import { StylelintConfig } from './types';
+
+// Success: Writes warnings to stdout
```

Comment:
Have you verified these?

Reply:
sure didn't

Reply:
I was just copying eslint, and figured it would do the same.  After checking, warnings are written to stderr.  I did verify success/failre works correctly with warnings

Reply:
Yeah.. there's like no consistency from tools on this. Each one is different -_-